stdlib/std.ncl

```diff
@@ -1213,24 +1215,42 @@
         # Examples
 
         ```nickel
-        flip (fun x y => "%{x} %{y}") "world!" "Hello,"
+        std.function.flip (fun x y => "%{x} %{y}") "world!" "Hello,"
           => "Hello, world!"
         ```
       "%%
       = fun f x y => f y x,
 
+    const
+      : forall a b. a -> b -> a
+      | doc m%"
+        Takes a value and returns the constant function which always returns
+        this value. Same as `std.function.first`.
+
+        # Examples
+
+        ```nickel
+        let f = std.function.const 5 in
+        f 7
+          => 5
+        ```
+      "%
+      = fun x y => x,
+
     first
       : forall a b. a -> b -> a
       | doc m%"
-        Always returns the first argument, ignoring the second.
+        Always returns the first argument, ignoring the second. Same as
+        `std.function.const`.
 
         # Examples
 
         ```nickel
-        first 5 7 => 5
+        std.function.first 5 7
+          => 5
         ```
       "%
-      = const,
+      = fun x y => x,
```

Comment:
The definition is very small, and overall it's simpler to avoid recursive definition in the stdlib